python/mlx/nn/layers/activations.py

```diff
@@ -143,6 +143,52 @@ class Sigmoid(Module):
     pass
 
 
+def step(x: mx.array, threshold: float = 0.0):
```

Comment:
This is a non-standard activation (given it's derivative is 0 almost everywhere). What's the motivation to include it?